spec/lib/geocode_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe Geocode do
+  it 'normalises the postcode' do
+    expect(Geocoder).to receive(:coordinates).with('RG11GG, United Kingdom')
+
+    Geocode.call('rg1 1gg')
+  end
+
+  it 'returns the lat/long pair' do
+    VCR.use_cassette(:rg11gg) do
+      latitude, longitude = Geocode.call('RG11GG')
+
+      expect(latitude).to  be
+      expect(longitude).to be
```

Comment:
Yes, I get that. My point is: If you'd changed the `Geocode.call` implementation to something like:

```ruby
def call
 # (...)
  :cat, :dog
end
```

It wouldn't fail the test, even though the return values are no longer lat/long pairs.